src/components/side-panel/layer-panel/color-selector.tsx

```diff
@@ -27,8 +27,8 @@ import ColorPalette from './color-palette';
 import {StyledPanelDropdown} from 'components/common/styled-components';
 import onClickOutside from 'react-onclickoutside';
 import {ColorUI} from 'layers/layer-factory';
-import {ColorRange} from 'constants/color-ranges';
-import {NestedPartial, RGBColor} from 'reducers';
+import {ColorRange} from '@kepler.gl/constants/color-ranges';
```

Comment:
It should be only '@kepler.gl/constants' here?

Reply:
No, I added export of all other paths, so we now can keep imports like this and save some time on future isolations

Reply:
better to import it from `@kepler.gl/constants` so when we are building kepler from src in studio, we don't need to add additional path resolver to our webpack config